catalog/transformer/catalog-transformer-rtf/src/main/java/ddf/catalog/transformer/output/rtf/RtfTemplate.java

```diff
@@ -41,6 +48,10 @@
  */
 public class RtfTemplate {
 
+  private static final Logger LOGGER = LoggerFactory.getLogger(RtfTemplate.class);
+
+  private static final int MINIMUM_VALID_IMAGE_DATA_SIZE = 10;
```

Comment:
❓ Are you sure 10 bytes is the smallest size an image can be? I found [this](https://github.com/mathiasbynens/small) repo where the smallest valid jpeg they could make was 107 bytes. 

We could give ourselves a little bit more clearance so that this bug doesn't come up again

Reply:
10 bytes is the smallest valid size to be able to parse the image type (which is where the exception is being thrown because the library doesn't check the array size).

Reply:
Sounds good. After thinking about it a bit more I think increasing the amount would be fixing a problem that doesn't exist yet and doing so might introduce other bugs.